packages/react/src/components/pickers/PeoplePicker/PeoplePickerItems/PeoplePickerItem.tsx

```diff
@@ -50,7 +61,7 @@ export const PeoplePickerItemBase = (props: IPeoplePickerItemSelectedProps) => {
         id={itemId}
         onClick={onRemoveItem}
         disabled={disabled}
-        iconProps={{ iconName: 'Cancel', styles: { root: { fontSize: '12px' } } }}
+        iconProps={{ iconName, styles: { root: { fontSize: '12px' } } }}
```

Comment:
If you change the new prop to be `removeButtonIconProps` instead of just the icon name (see my comment in BasePicker.types.ts), you can change this to be:
```suggestion
            iconProps={removeButtonIconProps ?? { iconName: 'Cancel' }}
            styles={{ icon: { fontSize: '12px' } }}
```